cns/NetworkContainerContract.go

```diff
@@ -23,6 +23,7 @@ const (
 	RequestIPConfig                          = "/network/requestipconfig"
 	ReleaseIPConfig                          = "/network/releaseipconfig"
 	GetIPAddresses                           = "/debug/getipaddresses"
+	GetHTTPRestData                        = "/debug/getrestdata"
```

Comment:
The name `getrestdata` is a bit generic. If the purpose of this endpoint is to return ip state, maybe the endpoint can be named `getipstate` or if its for a more generic placeholder, just `state`. Thoughts @neaggarwMS? Formatting is also off. You can enable goimports on save if you are using Goland.

Reply:
Yes, for the CLI command I have used `getInMemory` after we decide on the end-point name, I will change this too.